src/pudl/constants.py

```diff
@@ -2382,6 +2413,7 @@
         'primary_purpose_naics_id': pd.Int64Dtype(),
         'prime_mover_code': pd.StringDtype(),
         'pv_current_flow_type': pd.CategoricalDtype(categories=['AC', 'DC']),
+        'reactive_power_output': float,
```

Comment:
What are the units on this value? Should we have a suffix on the column name?

Reply:
Based on metadata below it looks like maybe `reactive_power_output_mvar`?  Reactive power is weird. https://en.wikipedia.org/wiki/AC_power#Reactive_power

Reply:
yep, that looks right